connector_magento/components/backend_adapter.py

```diff
@@ -101,27 +47,84 @@ def location(self):
         return location
 
 
```

Comment:
@guewen I found it out and just made a PR on your branch. Could you please have a look ?